packages/client/test/net/peer/libp2ppeer.spec.ts

```diff
@@ -31,7 +32,7 @@ tape('[Libp2pPeer]', async (t) => {
   t.test('should connect to peer', async (t) => {
     const config = new Config({ loglevel: 'error' })
     const peer = new Libp2pPeer({ config })
-    peer.on('connected', () => {
+    config.events.on(Event.PEER_CONNECTED, () => {
```

Comment:
so if all peers are emitting on this central bus then the event needs to return the peer or peerId for comparison and return if it isn't the right peer